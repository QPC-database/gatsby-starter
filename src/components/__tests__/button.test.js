import React from 'react'
import styled from 'react-emotion'
import renderer from 'react-test-renderer'

const Button = styled.div`
  color: hotpink;
`

test(`Button renders correctly -- proving that react-emotion and jest-emotion are working`, () => {
  expect(
    renderer.create(<Button>This is hotpink.</Button>).toJSON()
  ).toMatchSnapshot()
})
