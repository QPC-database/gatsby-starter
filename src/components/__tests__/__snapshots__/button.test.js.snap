// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly -- proving that react-emotion and jest-emotion are working 1`] = `
.emotion-0 {
  color: hotpink;
}

<div
  className="emotion-0 emotion-1"
>
  This is hotpink.
</div>
`;
